android/res/layout/localized_name.xml

```diff
@@ -41,10 +39,7 @@
     tools:drawableRight="@drawable/ic_expand_more"
     tools:ignore="UnusedAttribute"/>
 
-  <android.support.v7.widget.RecyclerView
-    android:id="@+id/names"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"/>
+  <include layout="@layout/recycler_default"/>
```

Comment:
Отлично подмечено, спасибо
> On Apr 29, 2016, at 11:39 AM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/res/layout/localized_name.xml <https://github.com/mapsme/omim/pull/3042#discussion_r61548114>:
> 
> > @@ -41,10 +39,7 @@
> >      tools:drawableRight="@drawable/ic_expand_more"
> >      tools:ignore="UnusedAttribute"/>
> >  
> > -  <android.support.v7.widget.RecyclerView
> > -    android:id="@+id/names"
> > -    android:layout_width="match_parent"
> > -    android:layout_height="wrap_content"/>
> > +  <include layout="@layout/recycler_default"/>
> Не забыл поменять id в коде?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3042/files/2cf6e1d2348df428db17da44048ac21121af3481..cb8c99cac36de3a210d65489c6571ec39042c078#r61548114>